downstream/modules/eda/con_rulebook-actions.adoc

```diff
@@ -0,0 +1,15 @@
+[id="rulebook-actions_{context}"]
+
+= Rulebook actions
+
+A rulebook specifies actions to be performed when a rule is triggered. A rule gets triggered when the events match the conditions for the rules. The following actions are currently supported:
+
+* `run_job_template`
+* `run_playbook`
```

Comment:
Should this action be removed if we are not officially supporting it in downstream? 
Please confirm @benthomasson @josephpisciotta

Reply:
@Alex-Izquierdo - I see your question has not been answered. I've posted a note in the #event-driven-ansible Slack channel and tagged Ben and Joe there with hope that we get an answer to this question soon.

Reply:
@Alex-Izquierdo - for now, I have commented out this line until we hear back from @benthomasson and @josephpisciotta .